pyproject.toml

```diff
@@ -223,3 +223,10 @@ concurrency = [
 
 [tool.coverage.html]
 show_contexts = true
+
+[tool.codespell]
+# Ref: https://github.com/codespell-project/codespell#using-a-config-file
+skip = '.git*,*.svg,*.lock'
+check-hidden = true
+ignore-regex = '\bTE\b'
```

Comment:
Can you add a comment for why this is needed (presumably you've found some intentional use of `TE`?)